devel/ldid-procursus/Portfile

```
@@ -28,6 +32,7 @@ universal_variant   no
 
 compiler.cxx_standard \
                     2011
+compiler.blacklist-append {clang < 1100}
 
```

Small nitpick, but otherwise, this looks good! Thanks :)

```suggestion
compiler.blacklist-append \
                    {clang < 1100}
```